app/models/external_usergroup.rb

```diff
@@ -37,14 +38,26 @@ def users
   private
 
   def in_auth_source?(source = auth_source)
-    errors.add :name, _("is not found in the authentication source") unless source.valid_group?(name)
-  rescue Net::LDAP::Error, Net::LDAP::LdapError => e
-    errors.add :auth_source_id, _("LDAP error - %{message}") % { :message => e.message }
+#    errors.add :name, _("is not found in the authentication source") unless source.valid_group?(name)
+#  rescue Net::LDAP::Error, Net::LDAP::LdapError => e
+#    errors.add :auth_source_id, _("LDAP error - %{message}") % { :message => e.message }
```

Comment:
I suppose this should not be commented

Reply:
Whoa - indeed, I did comment them to test in any auth source